utils/estimated_histogram.hh

```diff
@@ -73,40 +73,58 @@ struct estimated_histogram {
 
     int64_t _count = 0;
     int64_t _sample_sum = 0;
+    int64_t _min = 0;
+    int64_t _max = 0;
 
     estimated_histogram(int bucket_count = 90) {
 
         new_offsets(bucket_count);
         buckets.resize(bucket_offsets.size() + 1, 0);
     }
 
-    seastar::metrics::histogram get_histogram(size_t lower_bucket = 1, size_t max_buckets = 16) const {
+    /*
+     * \brief get an accumulative histogram
+     * When getting an accumulative histogram, you can set what is the first bucket to report
+     * what is the number of buckets and what is the ratio between bucket limits.
+     * Setting the bucket_ratio to 1 or less, will return all the buckets.
+     */
+    seastar::metrics::histogram get_histogram(size_t lower_bucket = 1, size_t max_buckets = 33, double bucket_ratio = 1.3) const {
```

Comment:
Why the change to 33? Not that I think there should be a default here (except maybe all).